code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/ExternalIdHandler.java

```diff
@@ -4,8 +4,8 @@
 
 public interface ExternalIdHandler {
 
-    public ExternalId transform(ExternalId externalId);
+    ExternalId transform(ExternalId externalId);
```

Comment:
Did we remove public because we want this interface to be available only within auth-logic package?